packages/cli/src/lib/bundler/config.ts

```diff
@@ -77,16 +79,65 @@ export function createConfig(
     }),
   );
 
+  const backendRelatedConfig = {
+    watch: true,
+    watchOptions: {
+      ignored: [/node_modules\/(?!\@backstage)/],
+    },
+    externals: [
+      nodeExternals({
+        modulesDir: paths.rootNodeModules,
+        whitelist: [
+          'webpack/hot/poll?100',
+          /\@backstage\/.*\/(?!node_modules)/,
+        ],
+      }),
+      nodeExternals({
+        modulesDir: paths.targetNodeModules,
+        whitelist: [
+          'webpack/hot/poll?100',
+          /\@backstage\/.*\/(?!node_modules)/,
+        ],
+      }),
+    ],
+    target: 'node' as const,
+    node: {
+      __dirname: true,
```

Comment:
Will this cause webpack to leave `__dirname ` intact instead of rewriting it? In that case we might not want it. Either way I don't think we can rely on `__dirname` at all in the future, as we're using a couple of different strategies for executing the code that doesn't guarantee any specific directory layout. Probably better to provide something similar to the `paths` utility in the cli where you can get the path of the package dir, whether it's in node_modules or inside the monorepo.

Reply:
I recall main problem was around the migrations as well, will check it out

Reply:
Moved to #1238